tests/end-to-end/batch/csharp/Stack.diff

```diff
@@ -1,13 +1,21 @@
 diff --git a/./tests/end-to-end/batch/template.json b/tests/end-to-end/batch-csharp-working-dir/cdk.out/Stack.template.json
-index c503353..684c8cb 100644
+index d75fe65..afb2161 100644
 --- a/./tests/end-to-end/batch/template.json
 +++ b/tests/end-to-end/batch-csharp-working-dir/cdk.out/Stack.template.json
-@@ -1,62 +1,5 @@
+@@ -1,69 +1,12 @@
```

Comment:
Why is did this diff change so much? Can we sort the JSON output first (in a separate PR) so that the diff becomes legible?